static/config/status-chain-genesis.json

```diff
@@ -0,0 +1,31 @@
+{
+  "alloc": {
+     "AdAf150b905Cf5E6A778E553E15A139B6618BbB7": {
+      "balance": "0x10F0CF064DD59200000"
+    },
+    "65C01586aa0Ce152835c788aCe665e91Ab3527b8": {
+      "balance": "0x1B1AE4D6E2EF500000"
+    }
+  },
+  "coinbase": "0x0000000000000000000000000000000000000000",
+  "config": {
+    "byzantiumBlock": 1700000,
+    "chainId": 777,
+    "daoForkSupport": true,
+    "eip150Block": 0,
+    "eip150Hash": "0x41941023680923e0fe4d74a34bdac8141f2540e3ae90623718e47d66d1ca4a2d",
+    "eip155Block": 10,
+    "eip158Block": 10,
+    "ethash": {},
+    "homesteadBlock": 0
+  },
```

Comment:
I may be wrong but let's try to only specify these values to be more consistent with real testnets:
```
"chainId": 777
"homesteadBlock": 0
"daoForkBlock": 0
"daoForkSupport": true
"byzantiumBlock": 0
"eip150Hash": <0 block hash>
"eip155Block": 0
"eip158Block": 0
```

Reply:
The genesis block I added is fully based on Ropsten's genesis block. (DefaultTestnetGenesisBlock). I can change it if necessary.

Reply:
Yes, please, let's try. Ropsten has already undergone those scheduled changes while our private network not. And it won't do it because we have no miners to mine blocks.